src/components/Dog.js

```diff
@@ -0,0 +1,5 @@
+import React from 'react'
+
+export default function Dog(){
```

Comment:
here, i think this should be a class based Component because this will eventually need state